src/sonic-py-common/sonic_py_common/general.py

```diff
@@ -1,4 +1,5 @@
 import sys
+import click
```

Comment:
>click [](http://example.com/codeflow?start=7&length=5)

Suggest not to depend on `click`. The user of the library has the freedom to use any framework other than `click`.